man/systemd.net-naming-scheme.xml

```diff
@@ -226,6 +231,10 @@
           <constant>v</constant> and the virtual device number, with any leading zeros removed. The bus
           number is ignored.</para>
 
+          <para>SR-IOV VF representors are named based on the name of the PF interface, with a suffix of
+          <constant>r</constant> and the number of the virtual device that is linked to the particular
+	  representor, with any leading zeros removed. The bus number is ignored.</para>
```

Comment:
Strange indentation here.

Based on the code, it seems that this suffix is used *instead* of a physical port name. Please describe this here too, this is quite important for the user.

Reply:
will fix